Collectors/DetailedCollector.py

```diff
@@ -228,9 +237,92 @@ def addRecord(self, sid, userID, fileClose, timestamp, addr, openTime):
+       elif fname.startswith('/hcc'):
+            return 'hcc'
+       elif fname.startswith('/gwdata'):   
+            return 'gwdata'
+       elif fname.startswith('/chtc/'):
+            return 'chtc'
+       elif fname.startswith('/icecube/'):
+            return 'icecube'
+       else:
+            return "noVO"
+
+
+    def process_gstream(self, gstream, addr, sid):                                                      
+
+        self.logger.info("GStream")
+        hostname = ""                                                                                   
+        hostip = ""                                                                                     
+        site = ""                                                                                       
+        lcg_record = False                                                                              
+        try:                                                                                            
+            hostip = addr                                                                               
+            try:                                                                                        
+                hostname = socket.gethostbyaddr(addr)[0]                                                
+            except Exception as e:                                                                      
+                self.logger.error("No able to gethostnyname")                                           
+                                                                                                        
+            if sid in self._servers:                                                                    
+                s = self._servers[sid]
+                if s .site is not None:
+                    site = s.site.decode('utf-8')
+                                                                                                        
+                                                                      
+            for event in gstream.events:
+                try: 
+                     event["sid"] = sid
+                     event["server_ip"] = hostip
+                     event["server_hostname"] = hostname
+                     event["file_path"] = event.pop("lfn")
+                     event["block_size"] = event.pop("blk_size")
+                     event["numbers_blocks"] = event.pop("n_blks")
+                     event["numbers_blocks_done"] = event.pop("n_blks_done")
+                     event["access_count"] = event.pop("access_cnt")
+                     event["attach_time"] = event.pop("attach_t")
+                     event["detach_time"] = event.pop("detach_t")
+                     remote = event.pop("remotes")
+                     if(len(remote) > 0):
+                          event["remotes_origin"] = remote
+                     else:
+                          event["remotes_origin"] = "null"
```

Comment:
Should this be "null" or just an empty string?

Reply:
"null" is easier to check in grafana

Reply:
I suppose.  But is "null" converted, or is it just a string of "null"?

Reply:
Maybe just empty string? `""`